src/stringProcessing/parseSynonyms.js

```diff
@@ -1,7 +1,7 @@
 /** @module stringProcessing/parseSynonyms */
 
 const stripSpaces = require( "../stringProcessing/stripSpaces.js" );
-const removePunctuation = require( "../stringProcessing/removePunctuation.js" );
+const removePunctuationExceptQuotes = require( "../stringProcessing/removePunctuationExceptQuotes.js" );
 
 /**
  * Matches strings from an array against a given text.
```

Comment:
Not really related to this PR, but I think the description of this function was probably copied from somewhere else and not adapted.